src/containers/trends/TrendsSVGContainer.js

```diff
@@ -41,13 +41,14 @@ const SMBG_OPTS = {
 
 import React, { PropTypes } from 'react';
 import dimensions from 'react-dimensions';
-
+import _ from 'lodash';
 
 import { MGDL_UNITS, MMOLL_UNITS } from '../../utils/constants';
 import { THREE_HRS } from '../../utils/datetime';
 import BackgroundWithTargetRange from '../../components/trends/common/BackgroundWithTargetRange';
 import CBGSlicesAnimationContainer from './CBGSlicesAnimationContainer';
 import SMBGRangeAvgAnimationContainer from './SMBGRangeAvgAnimationContainer';
+import { NoData, dataTypes } from '../../components/trends/common/NoData';
```

Comment:
I'd really prefer if for consistency with the way other components are done if you stick to what I initially advised and just put a `displayTypes` in the default props. Makes use of all our components consistent (not having to import multiple pieces).